cross/build-rootfs.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: $0 [BuildArch]"
+    echo "BuildArch can be: arm, arm64"
+
+    exit 1
+}
+
+__CrossDir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+__InitialDir=$PWD
+__BuildArch=arm
+__UbuntuArch=armhf
+__UbuntuRepo="http://ports.ubuntu.com/"
+__UbuntuPackages="build-essential libunwind8-dev gettext symlinks liblttng-ust-dev libicu-dev zlib1g-dev libssl-dev libcurl4-openssl-dev libz-dev libcrypto++-dev"
```

Comment:
The `build-rootfs.sh` is one time step you do to create the cross build environment. You want to have one cross build environment that works for all projects that you are working on (e.g. one for all coreclr/rt/fx). It is waste of disk space and pain to manage to have one per project. 

I think that it would best to move the scripts to create the shared cross build environment to https://github.com/dotnet/buildtools repo where we are maintaining build tools shared by different dotnet projects. (Or if it is not desirable to have it in buildtools for some reason, it can be a new repo with just the couple of files required to create the cross build environment.)